File: paradigmas de programação/lojinha.py
class exibir_produtos_do_estoque:
    def exibir_produtos_do_estoque():
        import estoque

class exibir_produtos_novos_do_estoque:
    def exibir_produtos_novos_do_estoque():
        import produtos_novos

class lojinha:
    print("\n************************************")
    print("***Bem-Vindo(a) à nossa lojinha!!***")
    print("************************************")
    print("\n////////////////////////////////////////////////////////////////////")
    print("///Esses são os produtos que constam no nosso estoque atualmente///")
    print("///////////////////////////////////////////////////////////////////")
    exibir_produtos_do_estoque.exibir_produtos_do_estoque()
    print("\n//////////////////////////////////////////////////////////////////")
    print("///Esses são os produtos novos que estão para chegar no estoque///")
    print("//////////////////////////////////////////////////////////////////")
    exibir_produtos_novos_do_estoque.exibir_produtos_novos_do_estoque()

File: paradigmas de programação/lojinha/lojinha.java
package lojinha;

class Lojinha {
    public static void main(String[] args) {
        System.out.println("\n************************************");
        System.out.println("***Bem-Vindo(a) à nossa lojinha!!***");
        System.out.println("************************************");
        System.out.println("\n////////////////////////////////////////////////////////////////////");
        System.out.println("///Esses são os produtos que constam no nosso estoque atualmente///");
        System.out.println("///////////////////////////////////////////////////////////////////");
        lojinha_produtos.ver_produtos();
        System.out.println("\n//////////////////////////////////////////////////////////////////");
        System.out.println("///Esses são os produtos novos que estão para chegar no estoque///");
        System.out.println("//////////////////////////////////////////////////////////////////");
        Lojinha_novos.ver_novos();
        
    }
}

